lib/api/test/unit/units/validateBody/getBodyValidator.test.js

```diff
@@ -0,0 +1,46 @@
+const { assert } = require('chai');
+const mediaTyper = require('media-typer');
+const { getBodyValidator } = require('../../../../units/validateBody');
+
+const getMediaTypes = (real, expected) => {
+  return [real, expected].map(mediaTyper.parse);
+};
+
+describe('getBodyValidator', () => {
+  describe('when given known media type', () => {
```

Comment:
what if given unknown media type?

Reply:
As we discussed previously, `getBodyValidator` is designed to work supported media types only. Preceding `getBodyType` returns only supported media types.

If we want to make sure `getBodyValidator` can digest any kind of media types, then I can suggest to use `TextDiff` as a fallback. However, as of now there is no need to do that, as it's not exposed publicly and internally always operates on supported media types.

Reply:
You're right, I'm sorry. I think I was misled by "when given known media type", which kinda implied there could be also the other case, so I pointed it out thinking about corner cases. Marking as resolved.